Applications/CLI/Tests.cmake

```diff
@@ -393,6 +393,113 @@ if(NOT OGS_USE_MPI)
     )
 
+    # SQUARE 1x1 GROUNDWATER FLOW TEST -- AXIALLY SYMMETRIC
+    # test results are compared to 3D simulation on a wedge-shaped domain
+    AddTest(
+        NAME GroundWaterFlowProcess_square_1x1_1e2_axi
+        PATH Elliptic/square_1x1_GroundWaterFlow
+        EXECUTABLE ogs
+        EXECUTABLE_ARGS square_1e2_axi.prj
+        WRAPPER time
+        TESTER vtkdiff
+        #ABSTOL 1.6e-3 RELTOL 1e-3
+        #DIFF_DATA
+        #wedge-1e2-ang-0.2-surface.vtu square_1e2_axi_pcs_0_ts_1_t_1.000000.vtu temperature temperature
+        ABSTOL 1.6e-5 RELTOL 1e-5
+        DIFF_DATA
+        wedge-1e2-ang-0.02-surface.vtu square_1e2_axi_pcs_0_ts_1_t_1.000000.vtu temperature temperature
+    )
+    # # WEDGE 1x1 GROUNDWATER FLOW TEST -- computes reference results for the above test
+    # AddTest(
+    #     NAME GroundWaterFlowProcess_wedge_1e2_ang_0.02
+    #     PATH Elliptic/square_1x1_GroundWaterFlow
+    #     EXECUTABLE ogs
+    #     EXECUTABLE_ARGS wedge_1e2_axi_ang_0.02.prj
+    # )
+
+    # SQUARE 1x1 GROUNDWATER FLOW TEST -- AXIALLY SYMMETRIC
+    # test results are compared to 3D simulation on a wedge-shaped domain
+    AddTest(
+        NAME GroundWaterFlowProcess_square_1x1_1e4_axi_ang_0.02
+        PATH Elliptic/square_1x1_GroundWaterFlow
+        EXECUTABLE ogs
+        EXECUTABLE_ARGS square_1e4_axi.prj
+        WRAPPER time
+        TESTER vtkdiff
+        ABSTOL 1.6e-5 RELTOL 1e-5
+        DIFF_DATA
+        wedge-1e4-ang-0.02-surface.vtu square_1e4_axi_pcs_0_ts_1_t_1.000000.vtu temperature temperature
+    )
+    # # WEDGE 1x1 GROUNDWATER FLOW TEST -- computes reference results for the above test
+    # AddTest(
+    #     NAME GroundWaterFlowProcess_wedge_1e4_ang_0.02
+    #     PATH Elliptic/square_1x1_GroundWaterFlow
+    #     EXECUTABLE ogs
+    #     EXECUTABLE_ARGS wedge_1e4_axi_ang_0.02.prj
+    # )
+
+    # SMALL DEFORMATION TEST -- AXIALLY SYMMETRIC
+    AddTest(
+        NAME SmallDeformation_ring_plane_strain_axi
+        PATH Mechanics/Linear
+        EXECUTABLE ogs
+        EXECUTABLE_ARGS ring_plane_strain.prj
+        WRAPPER time
+        TESTER vtkdiff
+        ABSTOL 6e-4 RELTOL 1e-4
+        DIFF_DATA
+        ring_plane_strain_1e4_solution.vtu ring_plane_strain_pcs_0_ts_1_t_1.000000.vtu u displacement
+        ring_plane_strain_1e4_solution.vtu ring_plane_strain_pcs_0_ts_1_t_1.000000.vtu sigma_rr sigma_rr
+        ring_plane_strain_1e4_solution.vtu ring_plane_strain_pcs_0_ts_1_t_1.000000.vtu sigma_ff sigma_ff
+    )
+
+    # SQUARE 1x1 HEAT CONDUCTION TEST -- AXIALLY SYMMETRIC
+    # test results are compared to 3D simulation on a wedge-shaped domain
+    AddTest(
+         NAME 2D_HeatConduction_axi
+         PATH Parabolic/T/2D_axially_symmetric
+         EXECUTABLE ogs
+         EXECUTABLE_ARGS square_1e2_axi.prj
+         WRAPPER time
+         TESTER vtkdiff
+         ABSTOL 1.7e-5 RELTOL 1e-5
```

Comment:
The reference results are from a simulation on a wedge-shaped domain having a wedge angle of 0.02 (rad). I assume that the difference is mostly caused by the size of the angle: Larger angle results in larger deviations.